ci/etcd.rb

```diff
@@ -36,7 +36,14 @@ def etcd_rootdir
 
     task :before_script => ['ci:common:before_script'] do
       sh %(cd $VOLATILE_DIR && #{etcd_rootdir}/etcd >/dev/null &)
-      sleep_for 10
+      # Waiting for etcd to start
+      Wait.for 'http://localhost:4001/v2/stats/self'
+      Wait.for 'http://localhost:4001/v2/stats/store'
+      10.times do
+        sh %(curl -s http://127.0.0.1:2379/v2/keys/message\
+             -XPUT -d value="Hello world" >/dev/null)
+        sh %(curl -s http://127.0.0.1:2379/v2/keys/message > /dev/null)
+      end
```

Comment:
I added it to generate some metrics which otherwise weren't always there (waiting for 10s did the trick previously, but not anymore :racehorse: )